css/jquery.multiselect.css

```diff
@@ -1,25 +1,29 @@
+/*  References:  
```

Comment:
I would prefer to stop including these reference comments. I was planning on going through and removing all of them from the old code once you were done merging changes.

I don't think they're really necessary and practices will change as web technologies evolve anyway.

Reply:
Ok